app/templates/play/modal/item-details-view.jade

```diff
@@ -41,8 +41,9 @@ if item && !item.owned
   if item.unequippable
     button.btn.big-font.disabled.unequippable(data-i18n="inventory."+item.get('heroClass').toLowerCase()+"_only")
   else
-    button#selected-item-unlock-button.btn.big-font(data-item-id=item.id, class=(item.get('subscriber') && !me.isPremium() ? "subscribe-button" : "unlock-button"))
-      if item.get('subscriber') && !me.isPremium()
+    - var showSubscribe = item.get('subscriber') && !me.isPremium() && !me.isStudent() && !me.showHeroAndInventoryModalsToStudents()
```

Comment:
Curious about the showHeroAndInventoryModalsToStudents check here.  Seems irrelevant at this point, since we're already showing items to whomever is here, and just deciding whether they see an unsubscribe button.  I think we only need an `!me.isStudent()` check.

Reply:
Thanks for the review. Will reduce the check to just the `!me.isStudent()`.